dbfit-java/sqlserver/src/integration-test/resources/acceptancescripts-SqlServer.sql

```diff
@@ -82,6 +82,22 @@ SET ANSI_NULLS ON
 GO
 SET QUOTED_IDENTIFIER ON
 GO
+IF NOT EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[Increment_P]') AND type in (N'P', N'PC'))
+BEGIN
+EXEC dbo.sp_executesql @statement = N'CREATE PROCEDURE [dbo].[Increment_P]
+	@counter INT OUTPUT
+AS
+BEGIN
+	SET @counter = isnull(@counter, 1) + 1
```

Comment:
With SQL Server case sensitivity depends upon the collation settings and I think they can be specified at different levels too (there was a PR as part of 3.2.0 related to this). Our test DBs are defined to be case insensitive anyway.